README.md

```diff
@@ -1,9 +1,12 @@
 # TiFlow
 
+[![LICENSE](https://img.shields.io/github/license/pingcap/tiflow.svg)](https://github.com/pingcap/tiflow/blob/master/LICENSE)
+![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/pingcap/tiflow)
+![GitHub Release Date](https://img.shields.io/github/release-date/pingcap/tiflow)
+![GitHub go.mod Go version](https://img.shields.io/github/go-mod/go-version/pingcap/tiflow)
 [![Build Status](https://github.com/pingcap/tiflow/actions/workflows/check_and_build.yaml/badge.svg?branch=master)](https://github.com/pingcap/tiflow/actions/workflows/check_and_build.yaml?query=event%3Apush+branch%3Amaster)
 [![codecov](https://codecov.io/gh/pingcap/tiflow/branch/master/graph/badge.svg)](https://codecov.io/gh/pingcap/tiflow)
 [![Coverage Status](https://coveralls.io/repos/github/pingcap/tiflow/badge.svg)](https://coveralls.io/github/pingcap/tiflow)
```

Comment:
Will we keep both these two coverage report, one for unit test and one for integration test

Reply:
After EE fix problem of `Coverage Status`, we will check the result and determine one, maybe either of the two will be ok.